test/functorch/test_ops.py

```diff
@@ -2060,6 +2068,8 @@ def reference(primals, cotangents, primals_tangents, cotangents_tangents):
                 # TODO: implement batching rule
                 xfail("_batch_norm_with_update"),
                 xfail("native_dropout_backward"),
+                xfail("igamma"),
+                xfail("igammac"),
```

Comment:
Does this fail for the same reason as above?  e.g. `forward AD with aten::igamma_self_backward that does not support it` Could we also add the comment here?